HISTORY.md

```diff
@@ -0,0 +1,306 @@
+# History
+
+## v2.3.2
```

Comment:
```suggestion
## [v2.3.2](https://github.com/voxpupuli/puppet-wildfly/tree/v2.3.2) (2018-08-18)
```

Reply:
then github changelog generator will properly detect it. regenerate the changelog after this is done.